src/Dex.tsx

```diff
@@ -9,6 +9,11 @@ import { Pokemon } from "./pkmn";
 
 const PAGE_SIZE = 50;
 
+var gChangeTab: (index: number) => void;
+var gupdateType0: (index: Type) => void;
+var gupdateType1: (index: Type) => void;
+var gupdateType2: (index: Type) => void;
```

Comment:
I'm not really sure what these are here for, or what "g" means

Reply:
g stands for global, since the makePKMN and makeType function can't access these functions unless they needlessly get passed through.

Reply:
I would prefer if you passed them through rather than using global variables 😃

Reply:
I will try and find a way :p

Reply:
No worries. If you get totally stuck later I could always help get it over the finish line.